frontend/Ingredishare/app/src/main/java/com/example/andriod/ingredishare/IngredientListActivity.java

```diff
@@ -7,13 +7,6 @@
 import android.location.Location;
 import android.location.LocationManager;
 import android.os.Bundle;
-import android.support.v4.app.ActivityCompat;
```

Comment:
import statements just changed from `android.support.v#...` to `androidx.appcompat...`